--- a/schema.rs
+++ b/schema.rs
@@ -9,9 +9,6 @@
     #[diesel(postgres_type(name = "recipient_type"))]
     pub struct RecipientType;
 
-    #[derive(diesel::query_builder::QueryId, diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "tsvector", schema = "pg_catalog"))]
-    pub struct Tsvector;
 }
 
 diesel::table! {
@@ -68,7 +65,7 @@
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;
 
     event_instances (post_id) {
         event_id -> Int8,
@@ -80,7 +77,7 @@
         created_at -> Timestamp,
         updated_at -> Nullable<Timestamp>,
         sync_source_instance_id -> Nullable<Varchar>,
-        search_text -> Tsvector,
+        search_text -> TsVector,
         user_id -> Nullable<Int8>,
         sync_missing_since -> Nullable<Timestamp>,
     }
@@ -245,7 +242,7 @@
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;
 
     messages (id) {
         id -> Int8,
@@ -256,7 +253,7 @@
         email_message_id -> Nullable<Varchar>,
         email_minio_path -> Nullable<Varchar>,
         created_at -> Timestamp,
-        search_text -> Tsvector,
+        search_text -> TsVector,
         messaging_group_id -> Int8,
     }
 }
@@ -282,7 +279,7 @@
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;
     use super::sql_types::PostMediaLayout;
 
     posts (id) {
@@ -307,7 +304,7 @@
         published_at -> Nullable<Timestamp>,
         last_activity_at -> Timestamp,
         unauthenticated_star_count -> Int8,
-        search_text -> Tsvector,
+        search_text -> TsVector,
         sort_published_at -> Timestamp,
         post_media_layout -> PostMediaLayout,
     }
@@ -399,7 +396,7 @@
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;
 
     users (id) {
         id -> Int8,
@@ -422,7 +419,7 @@
         response_count -> Int4,
         created_at -> Timestamp,
         updated_at -> Timestamp,
-        search_text -> Nullable<Tsvector>,
+        search_text -> TsVector,
         friend_count -> Int4,
         event_instance_count -> Int4,
     }
