--- a/schema.rs
+++ b/schema.rs
@@ -1,11 +1,5 @@
 // @generated automatically by Diesel CLI.

-pub mod sql_types {
-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "tsvector", schema = "pg_catalog"))]
-    pub struct Tsvector;
-}
-
 diesel::table! {
     event_attendances (id) {
         id -> Int8,
@@ -25,7 +19,7 @@

 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;

     event_instances (id) {
         id -> Int8,
@@ -38,7 +32,7 @@
         created_at -> Timestamp,
         updated_at -> Nullable<Timestamp>,
         event_sync_source_instance_id -> Nullable<Varchar>,
-        search_text -> Tsvector,
+        search_text -> TsVector,
         user_id -> Nullable<Int8>,
         sync_missing_since -> Nullable<Timestamp>,
     }
@@ -182,7 +176,7 @@

 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;

     posts (id) {
         id -> Int8,
@@ -206,7 +200,7 @@
         published_at -> Nullable<Timestamp>,
         last_activity_at -> Timestamp,
         unauthenticated_star_count -> Int8,
-        search_text -> Tsvector,
+        search_text -> TsVector,
         sort_published_at -> Timestamp,
     }
 }
@@ -293,7 +287,7 @@

 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;

     users (id) {
         id -> Int8,
@@ -316,7 +310,7 @@
         response_count -> Int4,
         created_at -> Timestamp,
         updated_at -> Timestamp,
-        search_text -> Nullable<Tsvector>,
+        search_text -> TsVector,
         friend_count -> Int4,
         event_instance_count -> Int4,
     }
