--- a/schema.rs	2026-08-07 08:22:37
+++ b/schema.rs	2026-08-07 08:24:03
@@ -4,10 +4,6 @@
     #[derive(diesel::query_builder::QueryId, diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "recipient_type"))]
     pub struct RecipientType;
-
-    #[derive(diesel::query_builder::QueryId, diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "tsvector", schema = "pg_catalog"))]
-    pub struct Tsvector;
 }
 
 diesel::table! {
@@ -29,7 +25,7 @@
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;
 
     event_instances (id) {
         id -> Int8,
@@ -42,7 +38,7 @@
         created_at -> Timestamp,
         updated_at -> Nullable<Timestamp>,
         event_sync_source_instance_id -> Nullable<Varchar>,
-        search_text -> Tsvector,
+        search_text -> TsVector,
         user_id -> Nullable<Int8>,
         sync_missing_since -> Nullable<Timestamp>,
     }
@@ -199,7 +195,7 @@
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;
 
     messages (id) {
         id -> Int8,
@@ -210,7 +206,7 @@
         email_message_id -> Nullable<Varchar>,
         email_minio_path -> Nullable<Varchar>,
         created_at -> Timestamp,
-        search_text -> Tsvector,
+        search_text -> TsVector,
         messaging_group_id -> Int8,
     }
 }
@@ -225,7 +221,7 @@
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;
 
     posts (id) {
         id -> Int8,
@@ -249,7 +245,7 @@
         published_at -> Nullable<Timestamp>,
         last_activity_at -> Timestamp,
         unauthenticated_star_count -> Int8,
-        search_text -> Tsvector,
+        search_text -> TsVector,
         sort_published_at -> Timestamp,
     }
 }
@@ -336,7 +332,7 @@
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Tsvector;
+    use diesel_full_text_search::TsVector;
 
     users (id) {
         id -> Int8,
@@ -359,7 +355,7 @@
         response_count -> Int4,
         created_at -> Timestamp,
         updated_at -> Timestamp,
-        search_text -> Nullable<Tsvector>,
+        search_text -> TsVector,
         friend_count -> Int4,
         event_instance_count -> Int4,
     }
